styles/Checkboxes.module.css

```diff
@@ -0,0 +1,21 @@
+.container {
+  font-size: 25px;
+  width: 100%;
```

Comment:
This width is making it so you can click really far away from the checkboxes and accidentally select one?